### ISSUE TITLE
Hyperparameters of GLUE datasets

### ISSUE THREAD

**Issue by mazicwong** [google-research/adapter-bert#6]
Hi, thanks for your great work!

I fail to reproduce the high results on the GLUE datasets.
Could you provide the hyperparameters w.r.t, training epoch, learning rate..., of the 9 tasks on the GLUE datasets corresponding to the best results claimed in the paper?
